### ISSUE TITLE
unexpected behaviors of `cut`/`copy`/`paste` bindings

### ISSUE THREAD

**Issue by chawyehsu** [okibcn/nano-for-windows#21]
cut/copy and paste will output garbled or invisible text.

example bindings
```
bind ^X cut main
bind ^C copy main
bind ^V paste main 
```

The same configuration works with nano-win.

**Comment by okibcn** [comment]
I can't duplicate the issue. I have that exact combination and works not only in Windows Terminal, but also in Wincon, though SSH from Windows or from a Linux system. Could you please describe the setup you are using? I can't do anything without a clear description of the environment.

**Comment by chawyehsu** [comment]
> Please note that ^V in Windows Terminal is captured by the terminal app, you have to unbind that key from WT to be usable in WT.

No, it does nothing with WT, not even work in PowerShell/ConsoleHost. Recorded in a brand new Windows 10 VM:

![nano](https://user-images.githubusercontent.com/5764917/222975976-f23f6753-5181-4324-8769-fdd36876a69f.gif)

**Comment by okibcn** [comment]
Could you please test pate with another key combo such as ^A or ^B, ^D in .nanorc?  Just to see whether the problem is the ^V capture or the underlying function.

I am working on integrating the clipboard with the windows clipboard which would eliminate this problem completely.

**Comment by chawyehsu** [comment]
> Could you please test pate with another key combo such as ^A or ^B, ^D in .nanorc? Just to see whether the problem is the ^V capture or the underlying function.

I just tested with `^A` and `^D`, and both worked.

**Comment by okibcn** [comment]
In the meantime, you can always disconnect the Ctrl-V combo from Windows Terminal, something required for other editors such as vim. Just read this: https://stackoverflow.com/a/65109836/7691477

Other terminal apps probably don't have the flexibility of overriding the windows key shortcuts. This is why I am implementing the windows clipboard within nano.

**Comment by chawyehsu** [comment]
> In the meantime, you can always disconnect the Ctrl-V combo from Windows Terminal, something required for other editors such as vim. Just read this: https://stackoverflow.com/a/65109836/7691477

Not my case, I don't have ^V combo for my Windows Terminal.
